### PR TITLE
07-routing-gossip.md: Describe standard human-readable format for short channel ID.

### PR DESCRIPTION
**Pull request by ZmnSCPxj**
Uncertain if this is best place to describe human-readable format, but, this section is what describes the components of the scid.

### CODE REVIEW

**Comment by cdecker** [comment]
ACK https://github.com/lightningnetwork/lightning-rfc/pull/506/commits/8a4e3fae4dc2646523cfa693d89d0d760023f71c

File: 07-routing-gossip.md [hunk 1]
```diff
@@ -57,6 +57,15 @@ It is constructed as follows:
...
   3. the least significant 2 bytes: indicating the output index that pays to the channel.
 
+The standard human readable format for `short_channel_id` is created
+by printing the above components, in the order:
+block height, transaction index, and output index.
+Each component is printed as a decimal number,
+and separated from each other by the small letter `x`.
+For example, a `short_channel_id` might be written as `539268x845x1`,
+indicating a channel on the output 1 of the transaction at index 845
+of the block at height 539268.
```
Comment (halseth) [inline 07-routing-gossip.md]: I'm not a native speaker, but should it be "index 845 *in* the block?

Comment (ZmnSCPxj) [reply 07-routing-gossip.md]: Construction "x of y" can mean "x is a part of y" or "x comes from y" or "x is owned by y", by my understanding of this language.  So we can consider "the transaction at index N of the block" as the transaction (which happens to be at index N) that is part of the block.

File: 07-routing-gossip.md [hunk 2]
```diff
@@ -76,6 +85,17 @@ A recipient node:
   - if it has sent AND received a valid `announcement_signatures` message:
     - SHOULD queue the `channel_announcement` message for its peers.
 
+#### Rationale
+
+The `short_channel_id` human readable format is designed
+so that double-clicking or double-tapping it will select the entire ID
+on most systems.
```
Comment (renepickhardt) [inline 07-routing-gossip.md]: ... on most operating systems ...

**Comment by Jagkree** [comment]
ดึงสิ

**Pull request merged**